ledfx/utils.py

```diff
@@ -355,11 +356,12 @@ async def get_config(self):
 
         wled_config = response.json()
 
-        if not wled_config["brand"] in "WLED":
-            msg = f"WLED {self.ip_address}: Not a compatible WLED brand '{wled_config['brand']}'"
-            raise ValueError(msg)
+        if "brand" not in wled_config:
+            raise ValueError(
+                f"WLED {self.ip_address}: Device is not WLED compatible"
+            )
 
-        _LOGGER.info(f"WLED {self.ip_address}: Received config")
+        _LOGGER.info(f"WLED {self.ip_address}: Received config {wled_config}")
```

Comment:
I don't think we need to drop the entire config here, since it's often huge.

Maybe "WLED compatible device (Brand: {wled_config['brand']) {self.ip_address} configuration received"?

Reply:
Ok, will trim down, test and merge.

Reply:
Now shows

WLED compatible device brand:www.athom.tech at 192.168.1.30 configuration received

This is for the controller that would not previous work. So now it just checks on the brand field existing, as a VERY sloppy test